regcore/migrations/0002_auto_20160308_2228.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import mptt.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('regcore', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='regulation',
+            name='children',
+        ),
```

Comment:
Sorry, I'm missing something, but when I access `reg.children` in the migration script, I get a related manager and not JSON, since I have `nested-sets-redux` checked out. The `children` column still exists in the database, but it's not accessible through the ORM, since the code in `models.py` doesn't know it exists. I'm going to ping you about this on chat to figure out what I'm missing here.